meddlr/metrics/build.py

```diff
@@ -54,7 +56,16 @@
     "DSC": (DSC, {}),
     "VOE": (VOE, {}),
     "ASSD": (ASSD, {"connectivity": 1}),
-    "CV": (CV, {})
+    "CV": (CV, {}),
+
+    # ======== Feature Based Metrics ============= #
+    "LPIPS": (LPIPS, {"net_type": "vgg",
+                      "mode": "grayscale",
+                      "lpips": True,
+                      "pretrained": True
+                      }),
```

Comment:
is this the default configuration used by LPIPS in some paper? if so add a comment with the paper link and the justification for these design choices. One clear justification is the use of grayscale instead of RGB because we deal predominantly with grayscale images